plugin/src/main/scala/com/softwaremill/clippy/AdviceLoader.scala

```diff
@@ -10,20 +11,34 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.io.Source
 import scala.tools.nsc.Global
 import scala.util.{Failure, Success, Try}
-import scala.collection.JavaConversions._
+import scala.collection.JavaConverters._
 
 class AdviceLoader(global: Global, url: String, localStoreDir: File, projectAdviceFile: Option[File])(implicit ec: ExecutionContext) {
   private val OneDayMillis = 1000L * 60 * 60 * 24
 
   private val localStore = new File(localStoreDir, "clippy.json.gz")
 
+  private def listFilesForFolder(folder: File): List[File] = {
+    folder.listFiles.flatMap { fileEntry =>
+      if (fileEntry.isDirectory)
+        listFilesForFolder(fileEntry)
+      else List(fileEntry)
+    }.toList
+  }
+
   private val resourcesAdvice: List[Advice] =
     getClass.getClassLoader
-      .getResources("clippy.json")
+      .getResources("")
+      .asScala
       .toList
-      .flatMap(loadAdviceFromUrL)
+      .flatMap { entry =>
+        val file = new File(entry.getFile)
+        listFilesForFolder(file)
```

Comment:
My intent was to load all files named *.clippy.json in any resources, so I thought it would also pull any out of jars.  For example my intention is that if there is a macwire.clippy.json in macwire when it is published, all users of clippy and macwire would get the advice.  Does this look like the correct way to do that to you?